src/api/uploads/MultiputUpload.js

```diff
@@ -0,0 +1,367 @@
+/**
+ * @flow
+ * @file Multiput upload
+ * @author Box
+ */
+
+import Base from '../Base';
+import { getFileLastModifiedAsISONoMSIfPossible } from '../../util/uploads';
+import createWorker from '../../util/uploadsSHA1Worker';
+import MultiputPart, { PART_STATE_UPLOADED, PART_STATE_DIGEST_READY } from './MultiputPart';
+import type { StringAnyMap } from '../../flowTypes';
+
+const DEFAULT_MULTIPUT_CONFIG = {
+    console: false,
+    digestReadahead: 5,
+    initialRetryDelayMs: 5000,
+    maxRetryDelayMs: 60000,
+    parallelism: 5,
+    partSize: null,
```

Comment:
Let's remove this config field as it is not publicly documented and we don't want people to start using it as it could change or go away.  Looks like you stopped sending it as a param to createSession already

Reply:
removed